modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -14,48 +14,73 @@ endif::[]
 |500 - Bad response from authorization server, or WSDL SOAP Fault error
 |===
 
-The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request.
+The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request. In Anypoint Platform, the client ID and client secret pair is also known as credentials.
```

Comment:
I would rephrase this a bit:

The Client ID Enforcement Policy restricts access to a protected resource by only allowing requests from registered client applications. The policy ensures that the client credentials sent on each request have been approved to consume the API.

When a client application is registered in Anypoint Platform, a set of credentials consisting of a client id and client secret pair are granted. Once the client application requests access to an API, a contract is generated between the application and the API. An API that is protected with a Client ID Enforcement policy will only be accessible by applications that have an approved contract. 

(In the see more section we link to all the client credentials plus contracts documentation)

Reply:
Great! Thanks!